data/transition-sites/dclg_fire.yml

```diff
@@ -0,0 +1,9 @@
+---
+site: dclg_fire
+whitehall_slug: department-for-communities-and-local-government
+homepage: https://www.gov.uk/government/policies/fire-prevention-and-rescue
+tna_timestamp: 20100105005659
+host: fire.gov.uk
+global: =301 https://www.gov.uk/government/policies/fire-prevention-and-rescue
+aliases:
+  - www.fire.gov.uk
```

Comment:
One more pedantic point: we [standardised a while ago](https://github.com/alphagov/transition-config/commit/872cf14b9ddd84ad2183a770b138cd2772cbe2d4) on no indentation in YAML in this repo. I can't remember exactly why we did that, but it would be good to stick to it for consistency if nothing else.